### PR TITLE
add validation of accepting only number to phone_number column

### PR DESCRIPTION
**Pull request by woshidan**
I would like to add validation of accepting only number to phone_number column in ActiveMerchant::Billing::ConvenienceStore.

### CODE REVIEW

File: test/unit/billing/convenience_store_test.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class EpsilonConvenienceStoreTest < MiniTest::Test
+  def test_blank_parameter_generate_error
```
Comment (hsbt) [inline test/unit/billing/convenience_store_test.rb]: It's great step for EpsilonConvenienceStore class testing.

File: test/fixtures/vcr_cassettes/find_sale_failure.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (kurotaky) [inline test/fixtures/vcr_cassettes/find_sale_failure.yml]: Is this necessary?

File: test/fixtures/vcr_cassettes/find_sale_success.yml [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
```
Comment (kurotaky) [inline test/fixtures/vcr_cassettes/find_sale_success.yml]: What is this fixtures file?

File: test/unit/billing/convenience_store_test.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+require 'test_helper'
+
+class EpsilonConvenienceStoreTest < MiniTest::Test
```
Comment (kurotaky) [inline test/unit/billing/convenience_store_test.rb]: Good!

**Comment by woshidan** [comment]
@kurotaky

Sorry. 

- test/fixtures/vcr_cassettes/find_sale_failure.yml
- test/fixtures/vcr_cassettes/find_sale_success.yml

I have commited these two files by mistake.
They have no relations to the function discussed in this pull request, 
So I have removed them.

**Comment by kurotaky** [comment]
@woshidan 
OK, I understood it.
Thank you for removing files :smile:

**Comment by kurotaky** [comment]
Release it! :rocket: 
* https://github.com/pepabo/active_merchant-epsilon/releases/tag/v0.5.4
* https://rubygems.org/gems/active_merchant-epsilon/versions/0.5.4

**Pull request merged**